.github/workflows/release.yml

```diff
@@ -54,4 +54,4 @@ jobs:
         working-directory: ./dist
         run: |
           npm pkg delete scripts.prepare
-          npm publish --access public --tag=next
```

Comment:
Why do we release a canary after we release a normal version

Reply:
This only works for prereleases because of the if
`if: "github.event.release.prerelease"`

Reply:
Ah didn't saw the `!` difference there, right